includes/class-simple-local-avatars.php

```diff
@@ -678,4 +716,158 @@ public function get_avatar_rest( $user ) {
 	public function set_avatar_rest( $input, $user ) {
 		$this->assign_new_user_avatar( $input['media_id'], $user->ID );
 	}
+
+	/**
+	 * Migrate the user's avatar data from WP User Avatar/ProfilePress
+	 *
+	 * This function creates a new option in the wp_options table to store the processed user IDs
+	 * so that we can run this command multiple times without processing the same user over and over again.
+	 *
+	 * Credit to Philip John for the Gist
+	 *
+	 * @see https://gist.github.com/philipjohn/822d3521a95481f6ad7e118a7106fbc7
+	 *
+	 * @return int
+	 */
+	public function migrate_from_wp_user_avatar() {
+
+		global $wpdb;
+
+		$count = 0;
+		$sites = get_sites();
+
+		// Bail early if we don't find sites.
+		if ( empty( $sites ) ) {
+			return $count;
+		}
+
+		foreach ( $sites as $site ) {
+			// Get the blog ID to use in the meta key and user query.
+			$blog_id = isset( $site->blog_id ) ? $site->blog_id : 1;
+
+			// Get the name of the meta key for WP User Avatar.
+			$meta_key = $wpdb->get_blog_prefix( $blog_id ) . 'user_avatar';
+
+			// Get processed users from database.
+			$migrations      = get_option( 'simple_local_avatars_migrations', array() );
+			$processed_users = isset( $migrations['wp_user_avatar'] ) ? $migrations['wp_user_avatar'] : array();
```

Comment:
Can be replaced with `??` version.

Reply:
The [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php) is only available in PHP 7.0 and above. The [minimum supported version of PHP for this plugin is 5.3](https://github.com/10up/simple-local-avatars/blob/develop/simple-local-avatars.php#L8).

Reply:
Won't fix